### PR TITLE
Abstract

### PR DESCRIPTION
**Pull request by cdeil**
@joleroi @adonath - Can you please review this abstract for the Gammapy 2016 conference (https://www.mpi-hd.mpg.de/hd2016/pages/news.php)?

It's pretty generic to leave us freedom to decide what to put later.

I don't know if there is any chance for technical talks. If yes we could apply for a talk. If no, we could apply for a poster directly. Any thoughts on this? Maybe you can ask around (e.g. Frank Rieger)?

Probably it's not worth writing a proceeding for this, because the paper will follow shortly. But we can continue improving our getting started analysis examples and put them on this poster and then also in the paper.

### CODE REVIEW

**Comment by cdeil** [comment]
@adonath @joleroi @jouvinlea - Please review this now and comment.

I copied the text from https://gammapy.readthedocs.io/en/latest/about.html#overview so I think it's OK.

Abstract submission deadline is May 22 and we have to circulate this by email with all co-authors.
So we should make sure we agree on this in the coming days...

**Comment by JouvinLea** [comment]
It's fine for me!

**Comment by adonath** [comment]
Here's the one from ICRC 2015: https://github.com/gammasky/gammapy-icrc2015/blob/master/gammapy_abstract_icrc_2015.txt

But I actually like the new version better, so no comment from my side... :+1: to use it as is.

File: abstract.txt [hunk 1]
```diff
@@ -0,0 +1,14 @@
+Gammapy - A Python package for gamma-ray astronomy
+
+Johannes King, Lea Jouvin, Axel Donath, Christoph Deil, Dirk Lennarz, Ellis
+Owen, Jonathan Harris, Luigi Tibaldo, Manuel Paz Arribas, Olga Vorokh, Regis
+Terrier, Rolf Bühler, Stefan Klepser, Victor Zabalza, Brigitta Sipocz
+
+Gammapy is an open-source Python package for gamma-ray astronomy. It is an
+in-development affiliated package of Astropy that builds on the core scientific
+Python stack (Numpy, Scipy, Astropy) to provide tools to simulate and analyse
```
Comment (joleroi) [inline abstract.txt:9]: simulate? Is there any functionality to do that?

I would say: [...] is a package [...] that provides functionality to analysis gamma-ray data from IACTs [...] as well as other [...]

Comment (cdeil) [inline abstract.txt:9]: For simulations there's [fake](http://cxc.harvard.edu/sherpa/ahelp/fake.html) in Sherpa which is basically `np.poisson` of `npred`, but still a very powerful simulation tool. We need to add more and make simulation a priority for the coming months, it's very important for code and method verification and obviously for CTA.
So I'd like to keep this aspect in the Gammapy description.

**Pull request merged**